trinity/main.py

```diff
@@ -81,15 +81,15 @@
 PRECONFIGURED_NETWORKS = {MAINNET_NETWORK_ID, ROPSTEN_NETWORK_ID}
 
 
-TRINITY_HEADER = (
+TRINITY_HEADER = "\n".join((
     "\n"
-    "      ______     _       _ __       \n"
-    "     /_  __/____(_)___  (_) /___  __\n"
-    "      / / / ___/ / __ \/ / __/ / / /\n"
-    "     / / / /  / / / / / / /_/ /_/ / \n"
-    "    /_/ /_/  /_/_/ /_/_/\__/\__, /  \n"
-    "                           /____/   "
-)
+    r"      ______     _       _ __       ",
```

Comment:
The PR appears to add an extra `\n` above the header, compared to the pre-diff. I don't really care, just making sure it was intentional.

Reply:
It was intentional